core/new-gui/src/app/workspace/component/result-panel/result-panel.component.ts

```diff
@@ -64,7 +77,27 @@ export class ResultPanelComponent {
       value => this.showResultPanel = value,
     );
   }
-
+  handleHighLightOperator() : void {
```

Comment:
add `public` before it. I know `public` is by default, but we add them to be more explicit.